.github/workflows/upgrade-typescript-api.yml

```diff
@@ -4,25 +4,27 @@ on:
     inputs:
       spec_version:
         description: runtime spec version (ex. 1601)
-        required: true
+        required: false
```

Comment:
It is used in `DOCKER_TAG` and seems required

Reply:
Fixed